app/src/main/res/layout/data_fragment.xml

```diff
@@ -17,13 +17,13 @@
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="Data available for training "
+            android:text="Data collected "
             android:textSize="20sp" />
 
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="Only the recordings in the selected subirectory 📂 of the current use case 💼 are considered" />
+            android:text="Only recordings in the current directory are considered" />
```

Comment:
Is it that obvious what is meant with `current directory`?

Reply:
yes